src/test/e2e/pageobjects/Search.page.ts

```diff
@@ -1,16 +1,9 @@
 import { SearchResultsPage } from './SearchResults.page';
+import {PageBase} from './Base/PageBase.page';
 
 const helpers = require('../Helpers/Selectors');
 
-export class SearchPage {
-
-  async getPageTitle(): Promise<string> {
-    $(helpers.SearchTitle).catch(() => {
```

Comment:
getPageTitle here is different compared to PageBase (it used helpers.SearchTitle instead of helpers.CommonPageTitle). Can you please check if this causes regression?